kubernetes/src/main/java/io/kubernetes/client/openapi/ModelUtil.java

```diff
@@ -0,0 +1,70 @@
+package io.kubernetes.client.openapi;
```

Comment:
This should be in the `util/src/main/java/...` directory, this directory is all generated code.

Reply:
would rename the package path to `io.kubernetes.client.util.conversion.Jobs`

Reply:
ok, I didn't figure out project structure. I can move into util